### PR TITLE
complete alert tabs table links message 5 component

### PR DESCRIPTION
**Pull request by YibeiLi777i**

### CODE REVIEW

File: apps/www/registry/sydney/example/alert-animation.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  ExclamationTriangleIcon,
+  InfoCircledIcon,
+  RocketIcon,
+} from "@radix-ui/react-icons"
+import { motion } from "framer-motion"
+import { AlertCircle, Terminal } from "lucide-react"
+
+import { Alert, AlertDescription, AlertTitle } from "@/registry/sydney/ui/alert"
+
+// 定义动画变体对象
```
Comment (hunterxu-gh) [inline apps/www/registry/sydney/example/alert-animation.tsx]: Please do not attach non-English comments.

**Review by hunterxu-gh — CHANGES_REQUESTED** [review]
Please do not attach non-English comments. (apps/www/registry/sydney/example/alert-animation.tsx) and the same component in other example folders.

File: apps/www/registry/sydney/example/alert-animation.tsx (hunk 1 above)
Comment (YibeiLi777i) [reply apps/www/registry/sydney/example/alert-animation.tsx]: i will check my code right now

Comment (YibeiLi777i) [reply apps/www/registry/sydney/example/alert-animation.tsx]: Ive deleted these comments.Thank you !

**Pull request closed without merging**